airlock/views/request.py

```diff
@@ -604,26 +615,72 @@ def request_multiselect(request, request_id: str):
 
     if not multiform.is_valid():
         display_form_errors(request, multiform.errors)
-    else:
-        action = multiform.cleaned_data["action"]
-        if action != "withdraw_files":
-            raise Http404(f"Invalid action {action}")
+        url = multiform.cleaned_data["next_url"]
+        return HttpResponse(headers={"HX-Redirect": url})
+
+    action = multiform.cleaned_data["action"]
+    match action:
```

Comment:
This might be nicer split out into functions for withdraw and update, like we do in the workspace version

Reply:
good idea, will do